templates/contact-us/form-data.json

```diff
@@ -45,18 +45,18 @@
                   "id": "22-04",
                   "value": "22.04 LTS",
                   "label": "22.04 LTS"
-                },
-                {
-                  "type": "checkbox",
-                  "id": "20-04",
-                  "value": "20.04 LTS",
-                  "label": "20.04 LTS"
                 }
               ]
             },
             {
               "fieldTitle": "LTS out of standard support",
               "options": [
+                {
+                  "type": "checkbox",
+                  "id": "20-04",
+                  "value": "20.04 LTS",
+                  "label": "20.04 LTS"
+                },
```

Comment:
On first glance, this looks like it should've succeeded in moving 20.04 LTS to LTS out of standard support" - but in the [demo](https://ubuntu-com-15855.demos.haus/contact-us/form?product=pro) I still see 20.04 LTS in "LTS within standard support".

<img width="1066" height="349" alt="2025-12-02_16-22" src="https://github.com/user-attachments/assets/2203657b-eeed-4145-b0db-0697cc39df8d" />


I'll spin this up locally to double-check there isn't some environment fluke going on.